Anchorage.xcodeproj/project.pbxproj

```diff
@@ -1012,6 +1012,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "com.raizlabs.$(PRODUCT_NAME)";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SDKROOT = appletvos;
+				TARGETED_DEVICE_FAMILY = 3;
```

Comment:
This is a good change, and #nth we could also set it to `4` for the `Anchorage-macOS` target.

Reply:
[This](https://twitter.com/jamesthomson/status/601038695026847744?lang=en) says 4 is watchOS. Either way, it doesn’t seem to be needed. I only added `3` because I got a compile-time warning, and I get no such warning for the macOS build.

Reply:
Huh, Xcode punked me. The only reason I suggested this is because I set it to 4 and it shrinks the device list down to just "My Mac". ¯\_(ツ)_/¯